src/data-server/sbvr-server.js

```diff
@@ -121,26 +122,26 @@ export function setup(app, sbvrUtils, db) {
 							is_of__vocabulary: 'data',
 						},
 					},
-				}),
-			)
-			.then((/** @type { Array<{ [key: string]: any }> } */ result) => {
-				if (result.length === 0) {
-					throw new Error('No SE data model found');
-				}
-				const instance = result[0];
-				return sbvrUtils.executeModel(tx, {
-					apiRoot: instance.is_of__vocabulary,
-					modelText: instance.model_value,
+				})
+				.then((/** @type { Array<{ [key: string]: any }> } */ result) => {
+					if (result.length === 0) {
+						throw new Error('No SE data model found');
+					}
+					const instance = result[0];
+					return sbvrUtils.executeModel(tx, {
+						apiRoot: instance.is_of__vocabulary,
+						modelText: instance.model_value,
+					});
 				});
-			})
-			.then(() => isServerOnAir(true))
-			.catch(() => isServerOnAir(false));
+			await isServerOnAir(true);
+		} catch {
+			isServerOnAir(false);
```

Comment:
Don't we typically also need an await here?

Reply:
Strictly speaking the `await` isn't needed for either when passing a value, but I'll add it